bundledata.go

```diff
@@ -168,6 +173,13 @@ func (d *BundleData) UnmarshaledWithServices() bool {
 	return d.unmarshaledWithServices
 }
 
+// SaasSpec represents a single software as a service (SAAS) node.
+// This will be mapped to consuming of offers from a bundle deployment.
+type SaasSpec struct {
+	Source string `bson:",omitempty" json:",omitempty" yaml:",omitempty"`
+	URL    string `bson:",omitempty" json:",omitempty" yaml:",omitempty"`
+}
```

Comment:
I wonder whether we need to add the controller UUID here so the client can avoid any confusion caused by similarly named controllers from its local cache.

@mitechie thoughts?

Reply:
Let's punt it for the moment. There's use cases where this will be something reused "relate me to an offer called 'keystone' on this controller" that we don't want to prevent.